bot/exts/fun/uwu.py

```diff
@@ -178,19 +181,28 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
         await clean_content(fix_channel_mentions=True).convert(ctx, text)
 
         fun_cog: t.Optional[Fun] = ctx.bot.get_cog("Fun")
+
         if fun_cog:
```

Comment:
Currently this also causes the line for `await fun_cog._get_text_and_embed(ctx, text)` to log this due to text being empty:
```
| bot.exts.fun.fun | DEBUG | Input '...' is not a valid Discord Message
```
It might be better to not run the `await fun_cog._get_text_and_embed(ctx, text)` at all if an embed is already available via `embeds` (see suggestions below)